roles/setup_dbt7/tasks/validate_setup_dbt7.yml

```diff
@@ -1,45 +1,40 @@
 ---
-# validate dbt7 package
-- name: Set dbt7 package name
+# validate dbt7 db packages installation
+- name: Set dbt7_db_pkg_list
   ansible.builtin.set_fact:
-    dbt7_package: 'dbt7'
+    dbt7_db_pkg_list: [ 'bc', 'byacc', 'dejavu-fonts-common', 'flex', 'fontconfig', 'fuse', 'fuse-libs', 'gcc', 'make', 'patch', 'perf', 'psmisc', 'sysstat', 'tmux', 'unzip' ]
```

Comment:
not sure why the linter didn't catch this but there doesn't need to be whitespace after/before the brackets

Reply:
oops, will fix